license.go

```diff
@@ -48,9 +48,14 @@ type License struct {
 		Company string `json:"Company"`
 		Email   string `json:"Email"`
 	} `json:"licensee"`
+	// Add on codes that may occur in legacy licenses that don't have a plan yet.
+	// https://gitlab.com/gitlab-org/gitlab/-/blob/master/ee/app/models/license.rb
```

Comment:
Think we can leave out the comments...

Reply:
First line is from original code, second - one and only source of addon codes. Are you sure? Is it some overhead with it or something else?

Reply:
Ok... In general I try to prevent these kind of inline comments, bit it seems this one makes good sense 👍🏻